### PR TITLE
DOC-2181: add fix documentation for TINY-10139 to the release notes.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2181

Changes:
* add fix documentation for TINY-10139 to the release notes.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/6.8-release-notes.adoc [hunk 1]
```diff
@@ -276,8 +276,20 @@ waiting for confirmation on these new feature tickets.
...
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  extended_valid_elements: 'mycustomblock[id],mycustominline',
+  custom_elements: 'mycustomblock,~mycustominline' // Notice the ~ prefix to force a span element for the element
+});
+----
+
+{productname} 6.8, now, checks if the element is a wrapper element, so it is also considers elements contained in custom element tags.
```
Comment (lorenzo-pomili) [inline modules/ROOT/pages/6.8-release-notes.adoc:292]: the problem here wasn't that there were no check but that the check didn't consider the custom elements

**Review by lorenzo-pomili — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/6.8-release-notes.adoc (hunk 1 above)
Comment (kemister85) [reply modules/ROOT/pages/6.8-release-notes.adoc:292]:
```suggestion
{productname} 6.8, now also considers **custom elements** as part of its process when checking if the element is a wrapper element.
```

Comment (kemister85) [reply modules/ROOT/pages/6.8-release-notes.adoc:292]: @lorenzo-pomili Please advise if you are happy with the above suggestion.

Comment (lorenzo-pomili) [reply modules/ROOT/pages/6.8-release-notes.adoc]: it works for me

**Pull request merged**